people.md

```diff
@@ -468,6 +468,18 @@ title: People
     </div>
   </div>
 
+  <div class="card d-flex d-block">
+    <img class="card-img-top" src="https://avatars.githubusercontent.com/u/81824608?v=4" alt="Mark Swillus">
+    <div class="card-body">
+      <div class="card-title">Mark Swillus</div>
+      <p class="card-text">Socio-technical enablers/inhibitors of software testing</a></p>
```

Comment:
where does this </a> start? 😄
```suggestion
      <p class="card-text">Socio-technical enablers/inhibitors of software testing</p>
```

Reply:
oh no, thats a left-over of the copy paste I am afraid 🙈 
Fix is on its way